pkg/utils/func.go

```diff
@@ -0,0 +1,9 @@
+package utils
```

Comment:
I generally don't see much value in utils like this. It just doesn't get reused.

Reply:
I'd usually agree but when something like `map` isn't in the language, this seems like a necessity.